raiden-ts/src/transport/epics/webrtc.ts

```diff
@@ -401,7 +401,13 @@ function listenDataChannel(
               );
             }),
           ),
-        ).pipe(finalize(() => (dataChannel.close(), connection.close()))),
+        ).pipe(
+          finalize(() => {
+            dataChannel.close();
+            // FIXME: https://github.com/node-webrtc/node-webrtc/issues/636
+            // connection.close();
+          }),
+        ),
```

Comment:
`connection.close` was causing a segfault, which made tests which stopped a node (like BF1) fail saying the node didn't stop cleanly (non-zero return code). This may be the cause for e2e tests non-clean return codes as well.
This change shouldn't affect us much, closing only the dataChannel and dropping the reference of the connection should make it be garbage collected (which doesn't segfault).

Reply:
Thanks for explaining this!